atomicfu/src/jvmTest/kotlin/kotlinx/atomicfu/test/TopLevelGeneratedDeclarationsReflectionTest.kt

```diff
@@ -1,5 +1,6 @@
 package kotlinx.atomicfu.test
 
+import java.lang.reflect.Modifier
```

Comment:
Nit: Redundant import here (make sure you have imports set to `import with '*'` in IDE)